[2025-09-25T11:30:01.432+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-25T11:30:01.465+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_pipeline_orchestration.check_analytics_api scheduled__2025-09-25T11:15:00+00:00 [queued]>
[2025-09-25T11:30:01.498+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_pipeline_orchestration.check_analytics_api scheduled__2025-09-25T11:15:00+00:00 [queued]>
[2025-09-25T11:30:01.502+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-09-25T11:30:01.529+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): check_analytics_api> on 2025-09-25 11:15:00+00:00
[2025-09-25T11:30:01.553+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=9090) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-25T11:30:01.566+0000] {standard_task_runner.py:64} INFO - Started process 9096 to run task
[2025-09-25T11:30:01.566+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_pipeline_orchestration', 'check_analytics_api', 'scheduled__2025-09-25T11:15:00+00:00', '--job-id', '82', '--raw', '--subdir', 'DAGS_FOLDER/weather_pipeline_orchestration.py', '--cfg-path', '/tmp/tmpdmk73_s8']
[2025-09-25T11:30:01.573+0000] {standard_task_runner.py:91} INFO - Job 82: Subtask check_analytics_api
[2025-09-25T11:30:01.674+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_pipeline_orchestration.check_analytics_api scheduled__2025-09-25T11:15:00+00:00 [running]> on host 791fb6ea4f60
[2025-09-25T11:30:01.809+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_pipeline_orchestration' AIRFLOW_CTX_TASK_ID='check_analytics_api' AIRFLOW_CTX_EXECUTION_DATE='2025-09-25T11:15:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-25T11:15:00+00:00'
[2025-09-25T11:30:01.817+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-25T11:30:01.943+0000] {logging_mixin.py:188} INFO - âœ… Analytics API is healthy
[2025-09-25T11:30:01.946+0000] {python.py:237} INFO - Done. Returned value was: API OK
[2025-09-25T11:30:01.947+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-25T11:30:02.009+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_pipeline_orchestration, task_id=check_analytics_api, run_id=scheduled__2025-09-25T11:15:00+00:00, execution_date=20250925T111500, start_date=20250925T113001, end_date=20250925T113002
[2025-09-25T11:30:02.081+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-25T11:30:02.136+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-25T11:30:02.139+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
